### PR TITLE
Allow webpack 3 as peer, use webpack 3 as devDep

### PR DESCRIPTION
**Pull request by snyamathi**
@sokra @TheLarkInn 

| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| Breaking change?  | no
| New feature?      | no
| Deprecations?     | no
| Tests added?      | no
| Docs updated?     | no
| Fixed tickets     | None
| License           | MIT

Since Webpack 3 has been released, this package should be updated to allow Webpack 3 as a peer dependency.  Not really necessary, but I've also bumped the `devDependency` on Webpack to 3 as well.

The API between Webpack 2 and 3 seems to be unchanged, so I'm not aware of any differences that would need to be accounted for in this package, it should just be a loosening of the `peerDependency` that's required.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "tasks"
   ],
   "peerDependencies": {
-    "webpack": "^2.1.0-beta || ^2.2.0-rc || 2"
+    "webpack": "^2.1.0-beta || ^2.2.0-rc || 2 || 3"
```
Comment (snyamathi) [inline package.json]: I could also allow the release candidate versions here if I change that to `^3.0.0-rc` but I'm not sure that's necessary now that `3.0.0` is out.

**Comment by danez** [comment]
Thanks.

**Comment by guidobouman** [comment]
Can a patch release be made based on this PR? Otherwise this results in both webpack 2 & 3 being installed in `node_modules`. Also, when using a `peerDependency`, shouldn't the `devDependency` be removed?

**Comment by danez** [comment]
the dev-dependency is for the tests. I will release a patch version.

**Comment by guidobouman** [comment]
Thanks!

**Pull request merged**